networking/ovn-kubernetes-network-provider/about-ovn-kubernetes.adoc

```diff
@@ -0,0 +1,28 @@
+[id="learn-about-ovn-kubernetes"]
+= Learn about the OVN-Kubernetes Pod network provider
```

Comment:
If this assembly is learning about the OVN-Kubernetes Pod network provider, then I think you should have some intro text like you had for OpenShift SDN. Otherwise this is really just a list of the exposed metrics.

Reply:
@bergerhoffer, yes, I plan on adding some. I wanted to get this merged first, as it's been sitting on the shelf for far too long.